src/splitpane/tests/unit/SplitPane.ts

```diff
@@ -0,0 +1,308 @@
+		let called = false;
+
+		widget.setProperties({
+			onResize: () => called = true,
+			direction: Direction.column
+		});
+
+		widget.sendEvent('mousedown', {
+			eventInit: <MouseEventInit> {
+				clientY: 110
+			},
+			selector: ':nth-child(2)' /* this should be the divider */
+		});
+		widget.sendEvent('mousemove', {
+			eventInit: <MouseEventInit> {
+				clientY: 150
+			},
+			selector: ':nth-child(2)' /* this should be the divider */
+		});
+		widget.sendEvent('mouseup', {
+			selector: ':nth-child(2)' /* this should be the divider */
+		});
+
+		assert.isTrue(called);
+	},
+
+	'Touch move should call onResize for row'(this: any) {
+		if (!hasTouch) {
+			this.skip('Environment not support touch events');
+		}
+
+		let called = false;
+
+		widget.setProperties({
+			onResize: () => called = true,
+			direction: Direction.row,
+			size: 100
+		});
+
+		widget.sendEvent('touchstart', {
+			eventInit: <MouseEventInit> {
+				changedTouches: [{ clientX: 110 }]
+			},
+			selector: ':nth-child(2)' /* this should be the divider */
+		});
+		widget.sendEvent('touchmove', {
+			eventInit: <MouseEventInit> {
+				changedTouches: [{ clientX: 150 }]
+			},
+			selector: ':nth-child(2)' /* this should be the divider */
+		});
+		widget.sendEvent('touchend', {
+			selector: ':nth-child(2)' /* this should be the divider */
+		});
+
+		assert.isTrue(called);
+	},
+
+	'Touch move should call onResize for column'(this: any) {
+		if (!hasTouch) {
+			this.skip('Environment not support touch events');
+		}
+
+		let called = 0;
+
+		widget.setProperties({
+			onResize: () => called++,
+			direction: Direction.column
+		});
+
+		widget.sendEvent('touchstart', {
+			eventInit: <MouseEventInit> {
+				changedTouches: [{ clientY: 110 }]
+			},
+			selector: ':nth-child(2)' /* this should be the divider */
+		});
+		widget.sendEvent('touchmove', {
+			eventInit: <MouseEventInit> {
+				changedTouches: [{ clientY: 150 }]
+			},
+			selector: ':nth-child(2)' /* this should be the divider */
+		});
+		widget.sendEvent('touchend', {
+			selector: ':nth-child(2)' /* this should be the divider */
+		});
+		widget.sendEvent('touchstart', {
+			eventInit: <MouseEventInit> {
+				changedTouches: [{ clientY: 150 }]
+			},
+			selector: ':nth-child(2)' /* this should be the divider */
+		});
+		widget.sendEvent('touchmove', {
+			eventInit: <MouseEventInit> {
+				changedTouches: [{ clientY: 110 }]
+			},
+			selector: ':nth-child(2)' /* this should be the divider */
+		});
+		widget.sendEvent('touchend', {
+			selector: ':nth-child(2)' /* this should be the divider */
+		});
+
+		assert.strictEqual(called, 2);
+	},
+
+	'onResieze should be called'() {
+		let called = 0;
+		const splitPane = new SplitPane();
+
+		splitPane.__setProperties__({ onResize: () => called++ });
+
+		(<any> splitPane)._root = { offsetWidth: 50, offsetHeight: 50 };
+		(<any> splitPane)._onResize();
+
+		splitPane.__setProperties__({
+			direction: Direction.column,
+			size: 25
+		});
```

Comment:
This test verifies that `onResize` is called if a window resize causes the root resize. I updated the test to be a bit better.